targets/XPlatCppSdk/source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -287,4 +303,4 @@ namespace PlayFab
     }
 }
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
space ?

Reply:
Same issue as above. git doesn't seem to understand removing these additional lines.

---
In reply to: [261336313](https://github.com/PlayFab/SDKGenerator/pull/419#discussion_r261336313) [](ancestors = 261336313)